client/src/components/SentimentAnalysis/SentimentAnalysis.js

```diff
@@ -8,17 +8,25 @@ import { INPUT_LIST } from '../../constants/analysis_constants';
 import ReCAPTCHA from 'react-google-recaptcha';
 import tardigrade from '../../images/tardigrade.png';
 
-const SentimentAnalysis = () => {
+const DEFAULT_ANALYSIS = {
+  entities: [],
+  sentiment: { score: 0.0, magnitude: 0.0 },
+};
+
+const SentimentAnalysis = props => {
+  const locationState = props.location.state;
   const [isVerified, setIsVerified] = useState(false);
-  const [header, setHeader] = useState({
-    entities: [],
-    sentiment: { score: 0.0, magnitude: 0.0 },
-  });
+  const [header, setHeader] = useState(
+    locationState !== undefined
+      ? locationState.ad.headlineAnalysis
+      : DEFAULT_ANALYSIS
```

Comment:
I think this would be clearer on one line. As is the ternary operator and the colon are stacked on top and is harder to read.